README.md

```diff
@@ -47,5 +49,82 @@ There's also a http plugin I nabbed from the esbuild docs which is not really ne
 - `poly.browser.js`: This can be used with a script tag in a browser with the `type="module"`. The external dependency is resolved as a remote URL (e.g. `https://cdn.skypack.dev/dot-beat-time`)
 - `poly.esm.js`: An ESM version of the module with external dependencies referred to by local name (e.g. `dot-beat-time`)
 - `poly.deno.js`: This can be used in Deno. Local imports have been transformed to URL imports.
-- `poly.node.js`: This is used by Node when imported using `import`.
-- `poly.node.cjs.js`: This is used by Node when imported using `require`.
+- `poly.node.mjs`: This is used by Node when imported using `import`.
+- `poly.node.cjs`: This is used by Node when imported using `require`.
+
+### Comparing the outputs
+
+#### `poly.browser.bundled.js`
+
+* 3rd party dependencies: bundled
+* Local dependencies: inlined
+* Container: IIFE, making a global `Poly` for accessing it from other browser scripts
+* Index file used: `index.browser.ts`
+* To use it from HTML:
+```html
+    <script src="/dist/poly.browser.bundled.js"></script>
+    <script>Poly.time()</script>
+```
+
+#### `poly.browser.js`
+
+* 3rd party dependencies: bundled (TODO: should be using skypack URLs instead?)
+* Local dependencies: inlined
+* Container: ESM `export`
+* Index file used: `index.browser.ts`
+* To use it from HTML:
+```html
+    <script src="/dist/poly.browser.js" type="module"></script>
+    <script>
+        import {time} from '/dist/poly.browser.js'
+        time();
+    </script>
+```
+
+#### `poly.deno.js`
+
+* 3rd party dependencies: bundled (TODO: should be using skypack URLs instead?)
+* Local dependencies: inlined
+* Container: ESM `export`
+* Index file used: `index.deno.ts`
+* To use it from a Deno script:
+```js
+    import { time } from 'poly';
+    poly.time();
+```
+
+#### `poly.esm.js`
+
+* 3rd party dependencies: `import { now } from 'dot-beat-time'`
+* Local dependencies: inlined
+* Container: ESM `export`
+* Index file used: `index.browser.ts`
+* To use it from... TODO: what context is this used from?
+```js
+    import { time } from 'poly';
+    poly.time();
+```
+
+#### `poly.node.cjs`
+
+* 3rd party dependencies: `require('dot-beat-time')`
+* Local dependencies: inlined
+* Container: CJS `module.exports = {...}`
+* Index file used: `index.node.ts`
+* To use it from a node script:
+```js
+    let poly = require('poly');
+    poly.time();
+```
+
+#### `poly.node.mjs`
+
+* 3rd party dependencies: `import { now } from 'dot-beat-time'`
+* Local dependencies: inlined
+* Container: ESM `export`
+* Index file used: `index.node.ts`
+* To use it from a node script:
+```js
+    import { time } from 'poly';
+    poly.time();
```

Comment:
There'd be no `poly` namespace unless you did `import * as poly from from 'poly'`